core/templates/components/forms/custom-forms-directives/audio-file-uploader.component.html

```diff
@@ -7,7 +7,12 @@
   </form>
   <div class="license-warning">
     Please only upload audio files that you created yourself and that you are willing to license under the site's
-    <a href="/license" target="_blank" rel="noopener">license terms</a>.
+    <a href="/license"
```

Comment:
Just confirming, the href and routerlink are both needed right (in all the places)

Reply:
yes, routerLink has to be added along with all occurances of href. This is best done along with the migration of a page to router.